src/Host.h

```diff
@@ -377,7 +377,8 @@ class Host : public QObject
     bool mFORCE_MXP_NEGOTIATION_OFF;
     QSet<QChar> mDoubleClickIgnore;
     QPointer<QDockWidget> mpDockableMapWidget;
-    // Set from last page of profile preferences if the timer interval is less
+    bool mEnableTextAnalyzer;
+    // Set from profile preferences, if the timer interval is less
```

Comment:
Actually, this fixed an error introduced by https://github.com/Mudlet/Mudlet/commit/62e970540e37248b1333bdaf14badb1dc47f5c2e which was a revising commit to PR #1903 which moved the control introduced by THAT Pull-Request but did not update that comment!